### ISSUE TITLE
Pyperclip v1.8.1: KeyError: 'XDG_SESSION_TYPE'

### ISSUE THREAD

**Issue by DattatrayPayatu** [asweigart/pyperclip#193]
pyperclip v1.8.1 throw **KeyError: 'XDG_SESSION_TYPE'** error when used with CMD2 v1.3.0 and above. (**pyperclip v1.8.0 is working perfect**.)

```
Traceback (most recent call last):
  File "/usr/local/bin/expliot", line 5, in <module>
    from expliot.expliot import EfCli
  File "/usr/local/lib/python3.8/dist-packages/expliot/expliot.py", line 4, in <module>
    from expliot.core.ui.cli import Cli
  File "/usr/local/lib/python3.8/dist-packages/expliot/core/ui/cli/__init__.py", line 5, in <module>
    from cmd2 import Cmd, with_argument_list
  File "/usr/local/lib/python3.8/dist-packages/cmd2/__init__.py", line 30, in <module>
    from .cmd2 import Cmd
  File "/usr/local/lib/python3.8/dist-packages/cmd2/cmd2.py", line 73, in <module>
    from .clipboard import (
  File "/usr/local/lib/python3.8/dist-packages/cmd2/clipboard.py", line 14, in <module>
    _ = pyperclip.paste()
  File "/usr/local/lib/python3.8/dist-packages/pyperclip/__init__.py", line 680, in lazy_load_stub_paste
    copy, paste = determine_clipboard()
  File "/usr/local/lib/python3.8/dist-packages/pyperclip/__init__.py", line 568, in determine_clipboard
    os.environ["XDG_SESSION_TYPE"] == "wayland" and
  File "/usr/lib/python3.8/os.py", line 675, in __getitem__
    raise KeyError(key) from None
KeyError: 'XDG_SESSION_TYPE'
```

**Comment by tleonhardt** [comment]
This issue was fixed by #177 and the release of `pyperclip` version 1.8.2

**Comment by DattatrayPayatu** [comment]
pyperclip version 1.8.2 resolved the issue with ubuntu 20.04 and python 3.8.5
